Scripts/new_powerpoint_maker_auto.py

```diff
@@ -0,0 +1,164 @@
+import os, sys
+from bible_passage import bible_passage_auto
+from slide_builders import create_from_template, create_bulletin_slide, create_offering_slide, create_starting_slides, create_title_and_text_slide, create_title_slide, add_title_with_image_on_right, append_song_to_powerpoint
+from helpers import get_next_sunday_auto, kill_powerpoint, find_song_names, parse_roster_row
+import PIL
+from fuzzywuzzy import process
+from song_finder import fetch_lyrics_auto
+
+# Global variable to store relative path information
+scripts_folder = os.path.dirname(__file__)
+
+def main():
+    '''
+    Creates a powerpoint with the required structure for BCCC from scratch. 
+    The powerpoint will be saved in the folder "Complete slides" with a filename of the date which is selected by the user
+    
+    Powerpoints have the structure
+    Title
+    No phones
+    A number of worship songs
+    Bible reading intro
+    Bible verses
+    Announcements
+    Tithing details
+    Prayer points
+    Mingle time slide
+    '''
+
+    print("Starting auto-script...")
+    font_sizes_large = {'title': 40, 'song': 27, 'bible reading': 23, 'tithing': 16}
+    font_sizes_medium = {'title': 50, 'song': 33, 'bible reading': 32, 'tithing': 23}
+    font_sizes_small = {'title': 70, 'song': 53, 'bible reading': 43, 'tithing': 32}
+
+    complete_ppt, template_path = create_from_template()
+    roster_sheet_link = 'https://docs.google.com/spreadsheets/d/1vgvPxJTzr0o1MUUaGb5AJqG6-WQL1PLzHHrwkAZPjQg/edit?gid=0#gid=0'
+
+    # Get the file name of the newly created file
+    saved_file_name = get_next_sunday_auto()
+    spreadsheet_date = get_next_sunday_auto("%d-%b-2024") # TODO: Should be %d-%b-%Y but the year is always set to 2024 in the spreadsheet
```

Comment:
@pangwuu If you can fix up the dates in the roster sheet to use the correct year, this change should be included before being approved:
```suggestion
    spreadsheet_date = get_next_sunday_auto("%d-%b-%Y")
```